Sources/Commands/SwiftTestTool.swift

```diff
@@ -145,6 +145,7 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
             throw TestError.multipleTestProducts
         } else {
             return buildPath
+                .appending(component: try getToolchain().destination.target)
```

Comment:
This whole area sucks, we should be able to use build plan to get the path now. This is fine for now, but if you're interested to clean this up, that'll be great.